### ISSUE TITLE
Using in replacement for entity-attributes-card

### ISSUE THREAD

**Issue by IanAdd** [benct/lovelace-multiple-entity-row#52]
The entity-attributes-card worked well for me when fleshing out details of each media_player. Now that looks like it's gone I am exploring this feature to get at attributes. The multi attributes per line is not helpful in this case as the attributes are words so I have used one line per attribute.
I'd like to be able to compact the rows and hide the icon. I really don't want to have to create masses of sensors.
Any suggestions?
![media_player](https://user-images.githubusercontent.com/10492213/75935740-414f1180-5ed4-11ea-89bb-095c5c09c8a4.jpg)

**Comment by benct** [comment]
There is no proper way to hide the icon, but if you add an invalid icon (`icon: nothing`) it will kinda be hidden. The state can be hidden with `show_state: false`.

It is not (easily) possible to make the rows more compact either I'm afraid. You could maybe use [card-mod](https://github.com/thomasloven/lovelace-card-mod) to change the row styling to make it more compact?

**Comment by IanAdd** [comment]
Great suggestion on icon: nothing which works.
I've explored card-mod and can't get it to adjust the bit I want.
The state-badge height is what I want to adjust - as seen in image - entered by hand.
The two card-mod attempts I've made are missing the spot.
```
    style:
      .: |
        ha-card {
          height: 17px;
        }
      state-badge:
        $: |
          :host {
            height: 17px;
          }

```
 Any suggestions.
![card-mod](https://user-images.githubusercontent.com/10492213/76138063-c7d23300-603b-11ea-82ef-f7bb8396620d.jpg)
